static/src/rocketLogic.js

```diff
@@ -91,31 +131,13 @@ const moveRockets = (board, activeRockets) => {
         return { startPosition: { x, y }, endPosition: { x: endX, y: endY }, rocket }
     })
 
-    const newActiveRockets = movedRockets.filter((moveRocket) => {
-        const { endPosition } = moveRocket;
-        return isMoveOnTheBoard(board, endPosition)
-    }).map((moveRocket) => {
-        const { startPosition, endPosition: { x, y } } = moveRocket;
-        const hitRocket = board[y][x]?.rockets[0];
-        debugLog(moveRocket);
-
-        // TODO put a moving rocket on the board!
-        if (hitRocket) {
-            debugLog('Hit a rocket')
-            debugLog(hitRocket);
-            board[y][x].explosion = true;
-        } else {
-            board[y][x].rockets = [moveRocket.rocket];
-        }
-        // clean up old rocket position
-        board[startPosition.y][startPosition.x].rockets = [];
-
-        // TODO: here we could do additional work, like special rockets that go in two, three, directions, or don't fire at all
-
-        if (hitRocket) {
-            return lightRocket({x, y}, hitRocket);
-        }
-        return { initialPosition: { x, y }, rocket: moveRocket.rocket };
+    const rocketsOnBoard = movedRockets.filter((moveRocket) => {
+        const { startPosition } = moveRocket;
+        return isMoveOnTheBoard(board, startPosition)
```

Comment:
this is the big change to clean up rockets, we need to keep rockets moving, until they are off the board.